src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -90,10 +95,10 @@ public int execute(SqlScript script) {
         }
     }
 
-    private File getTmpWrapperScript(SchemaType schemaType, File workingDir) {
+    private File getTmpWrapperScript(SchemaType schemaType, File workingDir, boolean isCompileSchemas) {
```

Comment:
`isCompileSchemas` is confusing, rename it to `isSkipInvalidsCompilation`

Reply:
fixed